### ISSUE TITLE
train text detection

### ISSUE THREAD

**Issue by soheildousti** [PaddlePaddle/PaddleOCR#16284]
### 🔎 Search before asking

- [x] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [x] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [x] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

I want to train a Paddle model for text detection on my own data. What changes should I make in my_det_config.yml file? Also, in the txt file for both train and val, it is in this format {"transcription": "text", "points": [[301, 67], [360, 67], [360, 76], [301, 76]]} . Should there be text content inside the transcription or is it enough to specify text or image?
Can someone help step by step?

### 🏃‍♂️ Environment (运行环境)

requrments

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

Global:
  use_gpu: true
  use_amp: true
  amp_level: O1
  epoch_num: 50
  log_smooth_window: 20
  print_batch_step: 50
  save_model_dir: ./output/det_r50_vd/
  save_epoch_step: 10
  eval_batch_step: [0,2000]
  cal_metric_during_train: False
  pretrained_model: /app/pretrain_models/ResNet50_vd_ssld_pretrained
  checkpoints:
  save_inference_dir:
  use_visualdl: False
  infer_img: doc/imgs_en/img_10.jpg
  save_res_path: ./output/det_db/predicts_db.txt

Architecture:
  model_type: det
  algorithm: DB
  Transform:
  Backbone:
    name: ResNet_vd
    pretrained: /app/pretrain_models/ResNet50_vd_ssld_pretrained.pdparams
    layers: 50
  Neck:
    name: DBFPN
    out_channels: 256
  Head:
    name: DBHead
    k: 50

Loss:
  name: DBLoss
  balance_loss: true
  main_loss_type: DiceLoss
  alpha: 5
  beta: 10
  ohem_ratio: 3

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    learning_rate: 0.001
  regularizer:
    name: 'L2'
    factor: 0

PostProcess:
  name: DBPostProcess
  thresh: 0.20
  box_thresh: 0.35
  max_candidates: 2000
  unclip_ratio: 2.0
  use_dilation: True

Metric:
  name: DetMetric
  main_indicator: hmean

Train:
  dataset:
    name: SimpleDataSet
    data_dir: 
    label_file_list:
      - /train.txt
    ratio_list: [1.0]
    transforms:
      - DecodeImage:
          img_mode: BGR
          channel_first: False
      - DetLabelEncode:
      - IaaAugment:
          augmenter_args:
            - { 'type': Affine, 'args': { 'rotate': [-6, 6] } }
            - { 'type': Resize, 'args': { 'size': [0.8, 1.2] } }
      - EastRandomCropData:
          size: [512, 512]
          max_tries: 10
          keep_ratio: true
      - MakeBorderMap:
          shrink_ratio: 0.4
          thresh_min: 0.3
          thresh_max: 0.7
      - MakeShrinkMap:
          shrink_ratio: 0.4
          min_text_size: 3
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'threshold_map', 'threshold_mask', 'shrink_map', 'shrink_mask']

  loader:
    shuffle: True
    drop_last: False
    batch_size_per_card: 4
    num_workers: 4

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: 
    label_file_list:
      - val.txt

    transforms:
      - DecodeImage:
          img_mode: BGR
          channel_first: False
      - DetLabelEncode:
      - DetResizeForTest:
          image_shape: [736, 1280]
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'shape', 'polys', 'ignore_tags']

  loader:
    shuffle: False
    drop_last: True
    batch_size_per_card: 1
    num_workers: 4

**Comment by leo-q8** [comment]
@soheildousti The transcription content doesn’t matter for detection training - as long as it’s not “###”, the detection model will automatically ignore the actual transcription content.

**Comment by TingquanGao** [comment]
The issue has no response for a long time and will be closed. You can reopen or new another issue if are still confused.

---
_From Bot_
